powerstore/resource_volume_group_snapshot.go

```diff
@@ -76,13 +76,14 @@ func (r *resourceVGSnapshot) Schema(ctx context.Context, req resource.SchemaRequ
 				MarkdownDescription: "Description of the volume group snapshot.",
 			},
 			"expiration_timestamp": schema.StringAttribute{
-				Required:            true,
+				Optional:            true,
```

Comment:
Same comment as the volume snapshot resource. Is it possible to remove the timestamp once assigned?

Reply:
This will be handled in update of volume group snapshot